File: admin-portal-main/lib/timesheets/Timesheet.dart
class Timesheet {
  final String id;
  final String name;
  final String status;
  final String phone;
  final String email;
  final String weekEndDate;
  final String hours;
  final String client;
  final String frequency;

  const Timesheet(
      {required this.email,
      required this.id,
      required this.name,
      required this.status,
      required this.phone,
      required this.weekEndDate,
      required this.hours,
      required this.client,
      required this.frequency});
}

File: employeeservice/src/main/java/com/rrets/employeeservice/model/timesheet/Timesheet.java
package com.rrets.employeeservice.model.timesheet;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.Getter;
import lombok.Setter;

import javax.persistence.*;
import java.time.LocalDate;
import java.util.List;

@Getter
@Setter
@JsonIgnoreProperties(ignoreUnknown = true)
@JsonInclude(JsonInclude.Include.NON_NULL)
@Entity
@Table(name = "timesheet")
public class Timesheet {
    @Id
    @Column(name = "Date", nullable = false)
    private LocalDate date;

    public LocalDate getDate() {
        return date;
    }

    public void setDate(LocalDate date) {
        this.date = date;
    }

    private String clientName;
    private String hours;
    private String status;
    private String frequency;
    private String active;
    private String deleted;
@Transient
    private List<LocalDate> dates;
}